spec/draft-barnes-cfrg-4q.md

```diff
@@ -109,29 +112,35 @@ N = 0x29cbc14e5e0a72f05397829cbc14e5dfbd004dfe0f79992fb2540ec7768ce7
 
 # Curve Points
 
-Elements of GF(p) are represented as 16-octet little-endian integers.  An element
-x0 + x1*i of GF(p^2) is represented on the wire by the concatenation of the byte strings for
-x0 and x1. Implementations will use whatever internal representation they desire, but we will
-describe the operations on elements of GF(p^2) assuming that X=x0+x1*i, with each x0 and x1
-an element of GF(p).
+Elements a = (a0, a1, ... , a126)_2 in GF(p) are represented as 128-bit (16-octet) little-endian 
+integers by concatenating a || 0, i.e., elements of GF(p) are encoded as (a0, a1, ... , a126, 0)_2. 
```

Comment:
I find this line confuses the heck out of me. Which bit in a byte is the MSB, which one the LSB? Are there any devices where octets aren't 8 bits?